src/plugins/gui/scenes/SepaFormScene.tsx

```diff
@@ -61,21 +61,24 @@ export const SepaFormScene = React.memo((props: Props) => {
   )
 
   return (
-    <SceneWrapper hasNotifications>
+    <SceneWrapper hasTabs hasNotifications avoidKeyboard>
+      {() => (
+        <>
```

Comment:
What does this function wrapper do? I thought Sam fixed the `SceneWrapper` so the behavior was the same, no matter if you passed a function or not.

Assuming the behavior *does* differ, wouldn't the inner scene responsible for handling the safe area insets now? I don't see that code being added.

Reply:
Fine combed the styling with this in mind, basically just following implementors elsewhere.

Knowing what we know about `KeyboardAwareScrollView` and iOS, applied the same hack fix here to make sure we don't have any paste and dodgy autoscroll issues.